main/tests/server/src/com/google/refine/RefineTest.java

```diff
@@ -431,6 +434,29 @@ protected ButterflyModule getCoreModule() {
         return coreModule;
     }
 
+    /**
+     * Runs an operation on a project. If it's a long-running operation, its process is run in the main thread until
+     * completion.
+     * 
+     * @returns the duration of the operation in milliseconds
+     */
+    protected long runOperation(AbstractOperation operation, Project project) throws Exception {
+        long start = System.currentTimeMillis();
+        Process process = operation.createProcess(project, new Properties());
```

Comment:
Can we align with how the real code does it?

```suggestion
        Process process = operation.createProcess(project, ExpressionUtils.createBindings(Project project));
```

Reply:
Can you point to the place in the real code where we do this? I'd find it surprising, because `ExpressionUtils.createBindings` is creating the environment for evaluating expressions, which is something completely different from the creation of a `Process`.

Reply:
Oops!  I was confusing this with the variable bindings. As a matter of fact, it doesn't look the options are ever used at all, so we should probably just remove that parameter (separately).